### ISSUE TITLE
PlayTurn()

### ISSUE THREAD

**Issue by primaulia** [wdi-sg/tic-tac-toe-tester-dorkblue#1]
Why did you check this first?

```
if (turn % 2 !== 0) {
```

Isn't playTurn function suppose to just fill your game array?

**Comment by dorkblue** [comment]
yes. i checked turn % 2 to determine player turn. if start is turn = 0, player one will only play on even turns
